models/trigger_map.go

```diff
@@ -6,20 +6,33 @@ import (
 
 type TriggerMapModel []TriggerMapItemModel
 
-func (triggerMap TriggerMapModel) Validate(workflows, pipelines []string) ([]string, error) {
-	var warnings []string
-	for _, item := range triggerMap {
-		warns, err := item.Validate(workflows, pipelines)
-		warnings = append(warnings, warns...)
+func (triggerMap TriggerMapModel) Normalised() ([]TriggerMapItemModel, error) {
```

Comment:
nit: normised vs. normalized, we should stick to one spelling across `trigger_map.go` and `trigger_map_item.go`

Reply:
updated in: https://github.com/bitrise-io/bitrise/pull/932/commits/25fb27de36a252d6e5ec46584178551700d6db55